libdevcore/Log.cpp

```diff
@@ -187,9 +187,7 @@ string dev::getThreadName()
 
 void dev::setThreadName(string const& _n)
 {
-#if defined(__GLIBC__)
-	pthread_setname_np(pthread_self(), _n.c_str());
-#elif defined(__APPLE__)
+#if defined(__APPLE__)
```

Comment:
You mean like this? Or shall I just cut the pthread naming entirely for all platforms?